package.json

```diff
@@ -14,7 +14,7 @@
     "@emotion/styled": "^11.11.0",
     "@fontsource/roboto": "^5.0.8",
     "@mui/icons-material": "^5.14.9",
-    "@mui/material": "^5.14.10",
+    "@mui/material": "^5.14.13",
```

Comment:
you must be careful when updating dependencies it can cause some problems with the project, so better to discuss it before doing so